src/_constants/index.js

```diff
@@ -17,7 +18,9 @@ const CONSTANTS = KeyMirror({
   CREATE_MODEL_NUMBER_SUCCESS: true,
   CREATE_MODEL_NUMBER_FAILURE: true,
   UPDATE_TOAST_MESSAGE_CONTENT: true,
-  RESET_TOAST_MESSAGE_CONTENT: true
+  RESET_TOAST_MESSAGE_CONTENT: true,
+  CREATE_CATEGORY_FAILURE: true,
+  CREATE_CATEGORY_SUCCESS: true,
```

Comment:
Remove the trailing comma. Also while at it, please change the `comma-dangle` rule in `.eslintrc` to `"comma-dangle": ["error", "never"]`. This will flag any trailing comma while still in dev environment

Reply:
fixed and added.